integreat_cms/core/settings.py

```diff
@@ -1005,6 +1005,16 @@
     strtobool(os.environ.get("INTEGREAT_CMS_SUMM_AI_IS_INITIAL", "True"))
 )
 
+# Slugs of regions that prefer Plain German over Easy German
+SUMM_AI_PLAIN_GERMAN_REGIONS: Final[str | list[str] | None] = os.environ.get(
+    "INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS", []
+)
+
+# Path of MT bulk action
+BULK_MT_PATH: Final[str] = os.environ.get(
+    "INTEGREAT_CMS_BULK_MT_PATH", "machine-translate"
+)
+
```

Comment:
```suggestion
```

I wouldn't add a setting for that, since the path cannot change without a code update anyways